CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ Then paste output from that script here.
 -----------------------------------------------------------------
 
 
+## v4.0.6 - 2022-12-05
+
+### Bugfixes
+* Fix Re-try button in Admin panel for RQJobs - [#2304](https://github.com/carpentries/amy/pull/2304) by @pbanaszkiewicz
+
```

Comment:
Something's off, I see these changes in `develop` branch. Can you try rebasing this hotfix branch against `develop`?

Reply:
I think rebasing would cause similar problems for the PR against `main`, so I made a new branch and a new PR: #2324